pkg/controller/controller_test.go

```diff
@@ -39,7 +40,17 @@ var _ = Describe("Controller", func() {
 		expectError   bool
 	}
 
-	setUpInformer := func(obj *v1.PersistentVolumeClaim, op operation, pvcName string) {
+	setUpInformer := func(obj *v1.PersistentVolumeClaim, op operation, pvcName string, ns string) {
+		// build queue value of ns + "/" + pvcName if exists
+		var buf bytes.Buffer
+		if len(ns) > 0 {
+			buf.WriteString(ns)
+			buf.WriteString("/")
+			buf.WriteString(pvcName)
+		} else {
+			buf.WriteString(pvcName)
+		}
+
```

Comment:
I think this is simpler and more typical in k8s code (replaces use of _bytes_ and buf block above):
```
// build queue value of work queue to be "ns/name" or just "name" if ns is blank
queueKey := pvcName
if len(ns) > 0 {
     queueKey = fmt.Sprintf("%s/%s", ns, pvcName)
}